internal/daemon/daemon.go

```diff
@@ -351,9 +351,19 @@ func logit(handler http.Handler) http.Handler {
 	})
 }
 
+// Validate performs the checks needed to ensure that the Pebble daemon can be started.
+func (d *Daemon) Validate() error {
+	_, err := d.overlord.ServiceManager().Plan()
```

Comment:
I agree with Fred's earlier comment, in that this is doing logic twice, but also this is a bit limited in what it can do. So in a way, it does too much, and too little at the same time.

What might be nice is to have an actual drymode that starts the overlord and all the managers, so all the little areas have a chance of attempting to run, and then shut it all down. We'll of course need to have a way to inform the managers that they're running dry, so they don't actually start work that could change the system.

Happy to discuss ideas here or elsewhere before you spent too much time on an uncertain avenue.